tedpermission/src/main/java/com/gun0912/tedpermission/TedPermissionActivity.java

```diff
@@ -211,7 +212,7 @@ public void finish() {
         overridePendingTransition(0, 0);
     }
 
-    private void showRationaleDialog(final ArrayList<String> needPermissions) {
+    private void showRationaleDialog(final List<String> needPermissions) {
```

Comment:
Is there a reason to change this?
It is a private function and there is no connection point with the outside.

Reply:
It was just base on `program to interfaces, not implementations`.

Although this request changed the listener used by the` ArrayList` to `List`, it was consistent under the same principle.